compiler/one-cmds/tests/one-codegen_neg_004.test

```diff
@@ -21,7 +21,7 @@ filename="${filename_ext%.*}"
 
 trap_err_onexit()
 {
-  if grep -q "error: 'command' key is missing in the configuration file" "${filename}.log"; then
+  if grep -q "error: Not found the command for given backend" "${filename}.log"; then
```

Comment:
```suggestion
  if grep -q "error: Command for given backend not found" "${filename}.log"; then
```
or if you want "Not" in the front,
`Not found with the command for given background`